css/modules.css

```diff
@@ -0,0 +1,62 @@
+.site-header {
+  background-color: #1EA9F2;
+  position: relative;
+
+}
+.heading-group-h1 {
+  float: left;
+}
+.icon-menu {
+  position: absolute;
```

Comment:
Right now the hamburger menu overlaps the header on screen sizes below about 400px. You might want to look into repositioning it for small screen sizes.